index.js

```diff
@@ -0,0 +1,103 @@
+const express = require("express");
+const cors = require("cors");
+const bcrypt = require("bcryptjs");
+const helmet = require("helmet");
+const knex = require("knex");
+const knexConfig = require("./knexfile.js");
+const server = express();
+const session = require("express-session");
+const KnexSessionStore = require("connect-session-knex")(session);
+
+const db = knex(knexConfig.development);
+const sessionConfig = {
+	store: new KnexSessionStore({
+		tablename: "sessions",
+		sidfieldname: "sid",
+		knex: db,
+		createtable: true,
+		clearInterval: 1000 * 60 * 60
+	}),
+	secret: "some.will.know.where.to.go",
+	name: "Ooga-Booga",
+	httpOnly: true,
+	resave: false,
+	saveUninitialized: false,
+	cookie: {
+		secure: false,
+		maxAge: 1000 * 60 * 1
+	}
+};
+
+server.use(helmet());
+server.use(express.json());
+server.use(cors());
+server.use(session(sessionConfig));
+
+server.get("/", (req, res) => {
+	res.send("Its Alive!");
+});
+
+server.get("/logout", (req, res) => {
+	if (req.session) {
+		req.session.destroy(err => {
+			if (err) {
+				res.send("You cant leave!");
+			} else {
+				res.send("goodbye!");
+			}
+		});
+	} else {
+		res.send(500).send("No Session To Log Out From");
+	}
+});
+
+server.post("/api/register", (req, res) => {
+	const credentials = req.body;
+	const hash = bcrypt.hashSync(credentials.password, 15);
+	credentials.password = hash;
+	//Cookie made for session
+	req.session.username = credentials.username;
+
+	db("users")
+		.insert(credentials)
+		.then(ids => {
+			res.status(201).json({ id: ids[0] });
+		})
+		.catch(err => {
+			res.status(500).json({ error: "could not create user" });
+		});
+});
+
+server.post("/api/login", (req, res) => {
+	const creds = req.body;
+	db("users")
+		.where({ username: creds.username })
+		.first()
+		.then(user => {
+			if (user && bcrypt.compareSync(creds.password, user.password)) {
+				//Cookie made for session
+				req.session.username = user.username;
+				res.status(201).json({ welcome: user.username });
+			} else {
+				res
+					.status(500)
+					.json({ error: "Wrong Username and/or Password, please try again" });
```

Comment:
Make all of your formatting consistent. Line 80 and lines 82-84 should look the same since they are doing very similar things.